mlflow/data/artifact_dataset_sources.py

```diff
@@ -107,13 +107,19 @@ def uri(self):
         def _get_source_type() -> str:
             return source_type
 
-        def download(self) -> str:
+        def load(self, /, dst_path=None) -> str:
```

Comment:
The `/` (kwarg only separator) is required to satisfy pylint that this method signature is a valid inheritance from the base class `load()` method.

Reply:
See https://stackoverflow.com/a/66500842/11952869 for context